07-classes/03-inheritance/script.js

```diff
@@ -15,5 +15,31 @@
             return `${this.constructor.greeting}! I'm ${this.name}!`;
         }
     }
-    // your code here
-})();
+
+    class Cat extends Animal {// making the first class
+        constructor(name) {
+            super();// to call the main class animal 
+            this.name = name;
+            this.constructor.greeting = "Meow";
+        }
+    }
```

Comment:
Peux-tu retirer `this.constructor.greeting = "Meow";`, et, en-dehors de la class, ajouter `Cat.greeting = "Meow"` ?